sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryNestedRecordsIT.java

```diff
@@ -93,40 +88,6 @@ private static void runPipeline(Options options) throws Exception {
                 .fromQuery(options.getUnflattenableInput())
                 .withoutResultFlattening());
 
-    final PCollectionView<Iterable<TableRow>> flattenedView =
```

Comment:
I think this is okay, but I would just caution deleting this test without adding it anywhere else. Is there anything this was testing that is not being tested anymore? Otherwise LGTM

Reply:
We opted to avoid Testing BQ+SideInputs and instead test only one at a time.